tests/DebugBar/Tests/DataCollector/AggregatedCollectorTest.php

```diff
@@ -17,7 +17,7 @@ public function testAddCollector()
         $this->c->addCollector($c = new MockCollector());
         $this->assertContains($c, $this->c->getCollectors());
         $this->assertEquals($c, $this->c['mock']);
-        $this->assertTrue(isset($this->c['mock']));
+        $this->assertArrayHasKey('mock', $this->c);
```

Comment:
Using the `assertArrayHasKey` to assert the expected key is on the result array.